dev-tools/package_test.go

```diff
@@ -47,7 +47,7 @@ const (
 )
 
 var (
-	configFilePattern   = regexp.MustCompile(`.*beat\.yml`)
+	configFilePattern   = regexp.MustCompile(`.*beat\.yml|apm-server\.yml`)
```

Comment:
I see we are already prepared for apm 👍

Reply:
Yes, I have tested this with APM. I updated the description to include a link to the `magefile.go` I used which customizes their packaging to not include the reference YAML and use a customized readme.md.